kolibri/plugins/learn/assets/src/vue/breadcrumbs/index.vue

```diff
@@ -1,39 +1,88 @@
 <template>
 
-  <nav class="nav" role="navigation" aria-label="You are here:">
-    <span class="parent">
-      <a v-link="rootLink">Explore</a> 
-    </span>
-    <span class="parent" v-for="crumb in crumbs">
-      <a v-link="crumbLink(crumb.id)">{{ crumb.title }}</a> 
-    </span>
-    <span class="current">
-      <span class="visuallyhidden">Current: </span>
-        <!-- TODO: Get current topic title -->
-    </span>
-  </nav>
+  <div>
+    <nav class="nav" role="navigation" aria-label="You are here:">
+      <span class="parent full-breadcrumbs" v-if="pageMode === thePageMode.EXPLORE && pageName !== thePageNames.EXPLORE_CONTENT && !isRoot">
+        <a v-link="exploreRoot">Explore</a>
+      </span>
+      <span v-if="pageName === thePageNames.LEARN_CONTENT">
+        <a v-link="learnRoot">
+          <span class='sep'>
```

Comment:
Should these still be class 'sep' ? that name doesn't seem accurate for a back arrow